bambou/session.go

```diff
@@ -392,12 +431,12 @@ func (s *Session) CreateChild(parent Identifiable, child Identifiable) *Error {
 
 	buffer := &bytes.Buffer{}
 	if err := json.NewEncoder(buffer).Encode(child); err != nil {
-		return NewError(ErrorCodeJSONCannotEncode, err.Error())
+		return NewBambouError("", err.Error())
 	}
 
 	request, err := http.NewRequest("POST", url, buffer)
 	if err != nil {
-		return NewError(http.StatusBadRequest, err.Error())
+		return NewBambouError("", err.Error())
```

Comment:
Why are we not returning the bad request code as title? Not a big fan of empty titles